static/docs/commands-reference/import-url.md

```diff
@@ -119,195 +119,155 @@ downloaded file or directory from the external data source.
+To illustrate the examples we will be using the project explained in the
+[Get Started](/doc/get-started) section. With it, it's very easy to setup a
+playground for the examples below.
 
 <details>
 
-### Click and expand to setup the project
+### Click and expand to setup the sample project
 
-This step is optional, and you can run it only if you want to run this examples
-in your environment. First, you need to download the project:
-
-```dvc
-    $ git clone https://github.com/iterative/example-get-started
-```
-
-Second, let's install the requirements. But before we do that, we **strongly**
-recommend creating a virtual environment with
-[virtualenv](https://virtualenv.pypa.io/en/stable/) or a similar tool:
+Follow these instructions before each example if you actually want to try them
+on your system. First, download the project and move into it:
 
 ```dvc
+$ git clone https://github.com/iterative/example-get-started
 $ cd example-get-started
-$ virtualenv -p python3 .env
-$ source .env/bin/activate
 ```
 
-Now, we can install requirements for the project:
+Then run:
 
 ```dvc
-$ pip install -r requirements.txt
+$ git checkout 2-remote
+$ mkdir data
 ```
 
+After executing these commands you should have a blank workspace, just before
+the _Add Files_ step mentioned before.
+
 </details>
 
 ## Example: Tracking a remote file
 
-The [DVC getting started tutorial](/doc/get-started) demonstrates a simple
-pipeline. In the [Add Files step](/doc/get-started/add-files) we are told to
-download a file, then use `dvc add` to integrate it with the workspace.
-
-An advanced alternate way to initialize the _Getting Started_ workspace, is
-using `dvc import-url`:
+The _Get Started_ section demonstrates a simple pipeline. In the
+[Add Files](/doc/get-started/add-files) step we are shown how to download a
+file, then use `dvc add` to integrate it with the workspace.
 
-<details>
-
-### Click and expand to prepare the workspace
-
-This is needed to actually run the command below in case you are trying this
-example:
-
-```dvc
-$ git checkout 2-remote
-$ mkdir data
-```
-
-After executing these commands you should have an almost empty workspace.
-
-</details>
+An advanced alternate to initialize the _Get Started_ workspace, is using
+`dvc import-url`:
 
 ```dvc
 $ dvc import-url https://dvc.org/s3/get-started/data.xml data/data.xml
-
 Importing 'https://dvc.org/s3/get-started/data.xml' -> 'data/data.xml'
 [##############################] 100% data.xml
-Adding 'data/data.xml' to 'data/.gitignore'.
-Saving 'data/data.xml' to cache '.dvc/cache'.
-Saving information to 'data.xml.dvc'.
-```
-
-To track the changes with git run:
-
-```dvc
-$ git add data/.gitignore data.xml.dvc
+...
 ```
 
-> Note that it's possible to set up the other
-> [stages](/doc/commands-reference/run) from the _Getting Started_ example, but
-> since we don't need them for this example, we'll skip it.
+> Note that it's possible to set up the other stages from the _Get Started_
```

Comment:
is actually important to mention?

Reply:
Good point. Not important for the example. Removed it in 1a214ea as well.